network-api/app/networkapi/settings.py

```diff
@@ -113,6 +117,7 @@
     'adminsortable',
 
     # the network site
+    'networkapi',
```

Comment:
interesting... what does this do?

Reply:
loads the management command. I felt it didn't belong in any other single app.